server/application/application.go

```diff
@@ -1093,7 +1093,8 @@ func (s *Server) RunResourceAction(ctx context.Context, q *application.ResourceA
 		Version:      q.Version,
 		Group:        q.Group,
 	}
-	res, config, _, err := s.getAppResource(ctx, rbacpolicy.ActionOverride, resourceRequest)
+	actionRequest := fmt.Sprintf("%s/%s", rbacpolicy.ActionAction, q.Action)
+	res, config, _, err := s.getAppResource(ctx, actionRequest, resourceRequest)
```

Comment:
Thinking about this more, `action/<actionname>` is not specific enough. We may have the same action name "pause", which works for both a Deployment, Rollout, and CronJob object. Granting "pause" privileges to all object types is potentially undesirable. We need a convention for actions which reflect the finer grained rbac. For example:

`action/argoproj.io/Rollout/resume`

Reply:
Responded in https://github.com/argoproj/argo-cd/issues/2181

Reply:
As I understand, the convention being introduced is `action/argoproj.io/Rollout:resume` instead of `action/argoproj.io/Rollout/resume`. I feel the slash is more consistent with all our other RBAC as opposed to the colon.

For example, rbac rules may be:
`action/argoproj.io/Rollout/*`

Which seems better than:
`action/argoproj.io/Rollout:*`

Reply:
Switched convention back to `action/argoproj.io/Rollout/*`